backend/src/emissions/schema.py

```diff
@@ -814,6 +832,60 @@ def mutate(root, info, input=None):
             return SetWorkingGroup(user=user, success=success, errors=e)
 
 
+class RequestJoinWorkingGroup(graphene.Mutation):
+    """GraphQL mutation to request to join a working group"""
+
+    class Arguments:
+        """Assign input type"""
+
+        input = JoinRequestInput()
+
+    success = graphene.Boolean()
+    join_request = graphene.Field(WorkingGroupJoinRequestType)
+
+    @staticmethod
+    @login_required
+    def mutate(root, info, input=None):
+        """Process incoming data"""
+        user = info.context.user
+        success = True
```

Comment:
By doing this, don't we just always return a successful message, even if something goes wrong in the following code? So if we raise a GraphQLError in Line 858, do we still return success=True?

Reply:
hm, good point. If a user requests to join a group which does not exist it will raise a `GraphQLError`. But this one doesn't contain the 'success' paramater. So we actually don't really use the `success` parameter at the moment. 

``` json
{"errors":[{
    "message":"[\'\\u201c1234\\u201d is not a valid UUID.\']",
   "locations":[{"line":3,"column":11}],
   "path":["requestJoinWorkingGroup"]}], 
   "data":{"requestJoinWorkingGroup":null}}
```

Reply:
ah we actually do return `sucess=False` it in the `PlanTrip` endpoint. But I guess here it's like that.

Reply:
Okay, seems fine to me then ;)